bril-ocaml/lib/bril.ml

```diff
@@ -250,15 +248,13 @@ let to_string { funcs } =
     | Nop -> `Assoc [ ("op", `String "nop") ]
   in
   let of_func { name; args; ret_type; instrs; _ } =
-    `Assoc
-      [
-        ("name", `String name);
-        ( "args",
-          `List
-            (List.map args ~f:(fun (name, bril_type) ->
-                 `Assoc [ ("name", `String name); ("type", of_type bril_type) ])) );
-        ("type", Option.value_map ret_type ~default:`Null ~f:of_type);
-        ("instrs", `List (List.map instrs ~f:of_instr));
-      ]
+      `Assoc (
+        [ ("name", `String name);
+          ( "args",
+            `List
+              (List.map args ~f:(fun (name, bril_type) ->
+                   `Assoc [ ("name", `String name); ("type", of_type bril_type) ])) );
+          ("instrs", `List (List.map instrs ~f:of_instr));
+        ] @ (Option.value_map ret_type ~default:[] ~f:(fun t -> [of_ret_type t])))
```

Comment:
i think this is a typo, `s/of_ret_type/of_type/`

Reply:
just realized it would have to be `[("type", of_type t)]` -- i've included these fixes in #80